docs/Attribute.md

```diff
@@ -2,7 +2,9 @@
 
 ## Introduction
 
-A class that publishes a getter or setter for an instance variable invites client classes to become too intimate with its inner workings, and in particular with its representation of state.
+A class that publishes a getter or setter for an instance variable invites
+client classes to become too intimate with its inner workings, and in
```

Comment:
We should also mention here that we only check for setters, not getters. Otherwise this is confusing.